eth_defi/aave_v3/constants.py

```diff
@@ -0,0 +1,176 @@
+        },
+    ),
+    # Avalanche Mainnet (XXX TODO - add more tokens)
+    'avalanche': AaveNetwork(
+        name='Avalanche',
+        pool_address='0x794a61358D6845594F94dc1DB02A252b5b4814aD',
+        pool_created_at_block=11970506,  # https://snowtrace.io/tx/0x300a7d036cddf30aa239f75afa5770efa557fdaf7fc67b79d1f0a183df619377
+        token_contracts={
+            # Aave token contracts defined in the Avalanche network
+            'AAVE': AaveToken(token_address='0x63a72806098Bd3D9520cC43356dD78afe5D386D9', deposit_address='0xf329e36C7bF6E5E86ce2150875a84Ce77f477375', variable_borrow_address='0xE80761Ea617F66F96274eA5e8c37f03960ecC679', stable_borrow_address='0xfAeF6A702D15428E588d4C0614AEFb4348D83D48', token_created_at_block=2749886),  # https://snowtrace.io/address/0x63a72806098bd3d9520cc43356dd78afe5d386d9
+            'WETH': AaveToken(token_address='0x49D5c2BdFfac6CE2BFdB6640F4F80f226bc10bAB', deposit_address='0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8', variable_borrow_address='0x0c84331e39d6658Cd6e6b9ba04736cC4c4734351', stable_borrow_address='0xD8Ad37849950903571df17049516a5CD4cbE55F6', token_created_at_block=2749895),  # https://snowtrace.io/address/0x49D5c2BdFfac6CE2BFdB6640F4F80f226bc10bAB
+        },
+    ),
+    # Harmony Mainnet Shard 0 (XXX TODO - add more tokens)
+    'harmony': AaveNetwork(
+        name='Harmony',
+        pool_address='0x794a61358D6845594F94dc1DB02A252b5b4814aD',
+        pool_created_at_block=23930374,  # https://explorer.harmony.one/tx/0xd89653b18dfe7558efd23569acc3cd6c8f51ee6b4005fc4bd65f084cad9caae9
+        token_contracts={
+            # Aave token contracts defined in the Harmony network
+            'AAVE': AaveToken(token_address='0xcF323Aad9E522B93F11c352CaA519Ad0E14eB40F', deposit_address='0xf329e36C7bF6E5E86ce2150875a84Ce77f477375', variable_borrow_address='0xE80761Ea617F66F96274eA5e8c37f03960ecC679', stable_borrow_address='0xfAeF6A702D15428E588d4C0614AEFb4348D83D48', token_created_at_block=5419567),  # https://explorer.harmony.one/address/0xcF323Aad9E522B93F11c352CaA519Ad0E14eB40F
+            'WETH': AaveToken(token_address='0x6983D1E6DEf3690C4d616b13597A09e6193EA013', deposit_address='0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8', variable_borrow_address='0x0c84331e39d6658Cd6e6b9ba04736cC4c4734351', stable_borrow_address='0xD8Ad37849950903571df17049516a5CD4cbE55F6', token_created_at_block=7907933),  # https://explorer.harmony.one/address/0x6983D1E6DEf3690C4d616b13597A09e6193EA013
+        },
+    ),
+}
+
+AAVE_V3_DEPOSIT_ADDRESS_TOKENS = {}  # autofill later
+
+
+# Helper functions for reading JSON-RPC URLs and account addresses from an optional aave.json file.
+# If you use it (instead of entering the values interactively), it should look like this:
+# {
+#   "json_rpc_url": "https://address-to-your-json-rpc-server,
+#   "account_address": "address-of-your-account"
+# }
+
+
+def aave_v3_get_json_rpc_url() -> str | None:
```

Comment:
[PEP 604](https://peps.python.org/pep-0604/) has only become live in Python 3.10, while we currently support Python 3.9+.

Fortunately, we can get around this by postponing the evaluation of annotations:
```py
from __future__ import annotations
```

I like this better than, say, using `Optional[str]` or `Union[str, None)`, as the pipe syntax is more concise.